workflow-templates/daily-security-check.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Daily security check
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+  push:
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      # PHP checks
+      - name: Check for php composer project
+        id: check_composer
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "composer.lock"
+      - name: Run php local security checker
+        if: steps.check_composer.outputs.files_exists == 'true'
+        uses: symfonycorp/security-checker-action@v4
+        continue-on-error: true
+
+      # node checks
+      - name: Check for node project
+        id: check_node
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "yarn.lock"
+      - name: Setup node
+        if: steps.check_node.outputs.files_exists == 'true'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Yarn Audit
+        if: steps.check_node.outputs.files_exists == 'true'
+        run: yarn audit --level high --groups dependencies optionalDependencies
+        continue-on-error: true
```

Comment:
Usually we use Yarn for getting hold of Node packages. However, in EngineBlock we use NPM for this task. If you want to make this action ultra portable, you might want to add a second check on a `package.lock` file. And in that case perform an `npm audit`

Reply:
I've added an (untested) npm audit block.